Makefile

```diff
@@ -181,7 +181,7 @@ install:
 	$(INSTALL_PROGRAM) extras/container/heketi-start.sh \
 		$(DESTDIR)$(hekdir)/container/heketi-start.sh
 	$(INSTALL_DATA) extras/container/heketi.json \
-		$(DESTDIR)$(hekdir)/container/heketi.json
+		$(DESTDIR)/etc/heketi/heketi.json
```

Comment:
I'm surprised to see this change and it appears to be undoing a pattern I was trying to establish. The pattern was twofold: 1) that paths are override-able using variables  2) the files "installed" to the system are "examples" that can then be copied or linked by an RPM spec file or Dockerfile or what not.
I'd like to see this bit dropped but add a line to the dockerfile to copy the "/usr/share/heketi/container/heketi.json" to /etc/heketi/heketi.json and similarly copy the heket-start.sh to /usr/bin/

Reply:
Packaging is one of the most annoying tasks (IMHO). And having to do the same steps in many different locations is error-prone. I'll do the copy for `/etc/heketi/heketi.json`, but do not see a reason to copy `heketi-start.sh` instead of just using it from the existing location.

Reply:
That's a fine & good point. Would you be open to doing something more like:
```Makefile
heketi_start_path:=$(DESTDIR)$(hekdir)/container/heketi-start.sh
heketi_config_path:=$(DESTDIR)$(hekdir)/container/heketi.json
# ...
	$(INSTALL_PROGRAM) extras/container/heketi-start.sh $(heketi_start_path)		
	$(INSTALL_DATA) extras/container/heketi.json $(heketi_config_path)

```
And then you'd be able to set the _path variables as needed in "make install"?